skills_taxonomy_v2/examples/extract_skills.py

```diff
@@ -0,0 +1,210 @@
+"""
+This pipeline creates and extracts skills from an input list of job adverts.
+
+Prerequisites:
+- You have access to our S3 bucket or have the skills classifier pkl file stored locally
```

Comment:
do we have the model in our repo? we could add the pkl model to outputs so people can have access to it